Replace the deployment guide/getting started guides with a single Installation Guide (#200)
ethyca/fides: docs/fides/docs/getting_started/manual.md

---

> Fidesctl can be spun up locally without relying on Docker, but it is a bit more complicated. If you'd like something simpler, please see the [Getting Started with Fidesctl in Docker](docker.md) guide for the recommended setup experience.

Reviewer: ```suggestion
Fidesctl can be started locally without relying on Docker, but it is more complicated. If you'd like something simpler, see the [Getting Started with Fidesctl in Docker](docker.md) guide for recommendations.
```

> The guide for getting Fidesctl up and running locally is the same as is described in our [Fides Production Deployment Guide](../deployment.md), so please head over there and follow the steps to get going.

Reviewer: ```suggestion
The guide for getting Fidesctl running locally is the same as described in the [Fides Production Deployment Guide](../deployment.md), so please follow those directions to get started.
```

## Next Steps

> See the [Tutorial](../tutorial/overview.md) page for a step-by-step guide on setting up a Fides data privacy workflow.

Reviewer: This change is coming in a PR very soon 😄 
```suggestion
See the [Tutorial](../tutorial/index.md) page for a step-by-step guide on setting up a Fides data privacy workflow.
```
